pkg/kustomize/apply.go

```diff
@@ -42,38 +43,47 @@ func (s *Kustomizer) Run(ctx context.Context, ready chan<- struct{}, stopped cha
 	defer close(stopped)
 	defer close(ready)
 
-	for _, path := range s.paths {
-		s.ApplyKustomizationPath(path)
+	kustomizationPaths, err := s.cfg.Manifests.GetKustomizationPaths()
+	if err != nil {
+		return fmt.Errorf("Could not find kustomization paths: %s", err)
+	}
+
+	for _, path := range kustomizationPaths {
+		s.applyKustomizationPath(path)
 	}
 
 	return ctx.Err()
 }
 
-func (s *Kustomizer) ApplyKustomizationPath(path string) {
+func (s *Kustomizer) applyKustomizationPath(path string) {
 	kustomization := filepath.Join(path, "kustomization.yaml")
```

Comment:
Not sure if you think this is worth changing since we explicitly call out the `kustomization.yaml` file in the docs. Technically `yaml` and `yml` are both valid extensions so we might need to try to stat both to avoid needless customer mistakes. 

It seems both the `kustomize` bin and  `kubectl` subcommand try `kustomization.yaml`, `kustomization.yml` and `kustomization` file names.

Reply:
That's a good point. I'll open another ticket to address that, since we'll need to check in a couple of places.